Elections.h

```diff
@@ -0,0 +1,123 @@
+//
+// Created by smark on 11.11.2022.
+//
+
+#ifndef LAB5_ELECTIONS_H
+#define LAB5_ELECTIONS_H
+#include <string>
+#include <iostream>
+
+namespace E {
+    using namespace C;
+    using namespace std;
+    class Elections {
+    private:
+        Candidate cand1;
+        Candidate cand2;
+        Candidate cand3;
+        Candidate cand4;
+        Candidate cand5;
+    public:
+        void printAll();
+        void setCandidates();
+        void countRates();
+        void whoWins();
+        Elections();
+        ~Elections()=default;
+    };
+
+    void Elections::setCandidates() {
+        cout << "\nfor candidate 1:";
+        cand1.setSurname();
+        cand1.setVotes();
+        cout << "\nfor candidate 2:";
+        cand2.setSurname();
+        cand2.setVotes();
+        cout << "\nfor candidate 3:";
+        cand3.setSurname();
+        cand3.setVotes();
+        cout << "\nfor candidate 4:";
+        cand4.setSurname();
+        cand4.setVotes();
+        cout << "\nfor candidate 5:";
+        cand5.setSurname();
+        cand5.setVotes();
+    }
+
```

Comment:
Please change to

void Elections::set_candidates() {
        cout << "\nfor candidate 1:";
        cand1.set_last_name();
        cand1.set_votes();
        cout << "\nfor candidate 2:";
        cand2.set_last_name();
        cand2.set_votes();
        cout << "\nfor candidate 3:";
        cand3.set_last_name();
        cand3.set_votes();
        cout << "\nfor candidate 4:";
        cand4.set_last_name();
        cand4.set_votes();
        cout << "\nfor candidate 5:";
        cand5.set_last_name();
        cand5.set_votes();
    }